pdm.lock

```diff
@@ -426,6 +437,26 @@ files = [
     {file = "pprintpp-0.4.0.tar.gz", hash = "sha256:ea826108e2c7f49dc6d66c752973c3fc9749142a798d6b254e1e301cfdbc6403"},
 ]
 
+[[package]]
+name = "pycairo"
```

Comment:
We need a graphics library now? 🤨

Reply:
I think we may not need... I was following the docs for pygobject

Reply:
I think pygobject pulls this anyway...

Reply:
That's an interesting indirect dependency. So it goes.